src/modules/createEvent/CreateEvent.tsx

```diff
@@ -86,8 +86,27 @@ const CreateEvent = ({ navigation, route }: CreateEventProps) => {
 
   const schema = data['eventSchemas'][route.params.type.toLowerCase()];
 
+  function hasAllEventDetails(eventData: any) {
+    return (
+      eventData.date &&
+      eventData.description &&
+      eventData.distance &&
+      eventData.endTime &&
+      eventData.location &&
+      eventData.startTime &&
+      eventData.title &&
+      eventData.uniqueMeetLocation
+    );
+  }
+
   const createEvent = () => {
     const eventDataCopy = { ...fields };
+
+    if (!hasAllEventDetails(eventDataCopy)) {
+      alert('missing required event details');
```

Comment:
We should use the Toast component for all error messages, just replace `alert` with something like
```
Toast.show('Missing required event details', {
      duration: Toast.durations.LONG,
      backgroundColor: 'red',
      position: 20,
});
```
(plus the import ofc)